.travis.yml

```diff
@@ -22,19 +20,18 @@ env:
 
 matrix:
     include:
-        - php: 5.6
+        - php: 7.2
           env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
         - php: 7.3
           env: DEPENDENCIES=dev
     allow_failures:
         - php: 7.3
```

Comment:
7.3 should not be allowed to fail anymore (`env: DEPENDENCIES=dev` may need it, but not the normal 7.3 job)

Reply:
You're right.
Maybe we can add `php: 7.4snapshot ` and use `env: DEPENDENCIES=dev` on it?

Reply:
Done